.dockerignore

```diff
@@ -0,0 +1,11 @@
+# Ignore everything
+**
+
+# Allow files and directories
+!/target/graalvm-native-image/codacy-swiftlint
```

Comment:
Is this supposed to be swiftlint? Or was it a copy + paste problem?

Reply:
Fixed ✅